build.gradle

```diff
@@ -125,67 +133,68 @@ minecraft {
 	}
 }
 
-// prevent java 8's strict doclint for javadocs from failing builds
-if (JavaVersion.current().isJava8Compatible()) {
-	tasks.withType(Javadoc) {
+javadoc {
+	source = [sourceSets.main.allJava, sourceSets.api.allJava]
+	// prevent java 8's strict doclint for javadocs from failing builds
+	if (JavaVersion.current().isJava8Compatible()) {
```

Comment:
Someone should double check but I am pretty sure this check shouldn't be needed if the above comment about setting the java toolchain is done.

Reply:
you're right about this one.